src/matlab2tikz.m

```diff
@@ -3842,8 +3845,52 @@ function warnFacetedInterp(m2t)
     [m2t, drawOptions] = setColor(m2t, h, drawOptions, 'fill', faceColor);
 end
 % ==============================================================================
+function [m2t,str] = drawBaseline(m2t,hparent,isVertical)
+    str = '';
+    baseValue = get(hparent, 'BaseValue');
+    if isOff(get(hparent,'ShowBaseLine')) || ~isHG2() || baseValue ~= 0
+        return
+    end
+    
+    if ~exist('isVertical','var')
```

Comment:
Please put this at the top of the function, that makes sure that `isVertical` is available in the whole function.